README.md

```diff
@@ -14,14 +14,29 @@ Once you've created something amazing with Open MCT, showcase your work in our G
 Building and running Open MCT in your local dev environment is very easy. Be sure you have [Git](https://git-scm.com/downloads) and [Node.js](https://nodejs.org/) installed, then follow the directions below. Need additional information? Check out the [Getting Started](https://nasa.github.io/openmct/getting-started/) page on our website.
 (These instructions assume you are installing as a non-root user; developers have [reported issues](https://github.com/nasa/openmct/issues/1151) running these steps with root privileges.)
 
-1. Clone the source code `git clone https://github.com/nasa/openmct.git`
+1. Clone the source code 
+````
+git clone https://github.com/nasa/openmct.git
+```
+2. (Optionally) Install the correct node version using [nvm](https://github.com/nvm-sh/nvm) 
 
-2. (Optionally) Install the correct node version using [nvm](https://github.com/nvm-sh/nvm) (`nvm install`)
-3. Install development dependencies. Note: Check the package.json engine for our tested and supported node versions. `npm install`
+```
+nvm install
+```
 
-4. Run a local development server `npm start`
+3. Install development dependencies. Note: Check the package.json engine for our tested and supported node versions. 
 
-Open MCT is now running, and can be accessed by pointing a web browser at [http://localhost:8080/](http://localhost:8080/)
+```
+npm install
+```
+
+4. Run a local development server 
+```
+npm start
+```
+
+> [!IMPORTANT]
```

Comment:
This doesn't display correctly for me and I'm not familiar with the syntax. What Markdown viewer are you using and what effect is it supposed to have?

![image](https://github.com/nasa/openmct/assets/9259993/788c2b64-2f2b-4ed7-b491-4beabb2e7162)

Reply:
I was using chrome and the github fileviewer:

https://github.com/nasa/openmct/blob/28788490197bb2e75cba2c923caf344f3d4ad06b/README.md